Source/WebKit/UIProcess/API/Swift/WKUIDelegateAdapter.swift

```diff
@@ -35,7 +36,7 @@ private struct DefaultDialogPresenting: DialogPresenting {
 }
 
 @MainActor
-final class WKUIDelegateAdapter: NSObject, WKUIDelegate {
+final class WKUIDelegateAdapter: NSObject, WKUIDelegatePrivate {
```

Comment:
I think the commit message needs to explain why this isn't `WKUIDelegateInternal`.